pkg/doc/verifiable/test-suite/verifiable_suite_test.go

```diff
@@ -186,6 +187,39 @@ func parseKeys(packedKeys string) (private, public interface{}) {
 	return privateKey, publicKey
 }
 
+func getJWTDecodePublicKey() interface{} {
```

Comment:
I do not think we should be hard-coding the key in the code.

The key provided in the command-line arguments should be used.

Reply:
The thing is that  JWS from VC Test Suite are static - they are not re-build based on the keys provided to the Suite in `config.json`. We can actually customize the keys and they then will differ from the original key used to prepare static JWS in the Suite. That's why I want to emphasize that the key is hardcoded and does not depend on `config.json`.

I am thinking that we may probably propose to generate `example-016-jwt.jwt` (i.e. not to use hard-coded value) in the Suite to avoid such uncertainty.

Reply:
After discussion with @troyronda , this is made simpler - use the RSA key from `config.json` to verify the signature.